plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -504,6 +511,56 @@ public void testDeletingEntirePartitionedTable()
     }
 
+    @Test
+    public void testFilesTable()
+            throws Exception
+    {
+        String tableName = "test_files_table_" + randomNameSuffix();
+        String tableLocation = metastoreDir.getPath() + "/" + tableName;
+        assertUpdate("CREATE TABLE " + tableName + " WITH (location = '" + tableLocation + "', format_version = 2) AS SELECT * FROM tpch.tiny.nation", 25);
+        BaseTable table = loadTable(tableName);
+        Metrics metrics = new Metrics(
+                10L,
+                ImmutableMap.of(1, 2L, 2, 3L),
+                ImmutableMap.of(1, 5L, 2, 3L, 3, 2L),
+                ImmutableMap.of(1, 0L, 2, 2L),
+                ImmutableMap.of(4, 1L),
+                ImmutableMap.of(1, ByteBuffer.allocate(8).order(ByteOrder.LITTLE_ENDIAN).putLong(0, 0L)),
+                ImmutableMap.of(1, ByteBuffer.allocate(8).order(ByteOrder.LITTLE_ENDIAN).putLong(0, 4L)));
+        DataFile dataFile =
+                DataFiles.builder(PartitionSpec.unpartitioned())
+                        .withFormat(FileFormat.ORC)
+                        .withPath(tableLocation + "/data/20230310_133318_00004_qhftg-870c1a85-e56c-4536-9b84-33c69166ddfb.orc")
+                        .withFileSizeInBytes(1234)
+                        .withMetrics(metrics)
+                        .withSplitOffsets(ImmutableList.of(4L))
+                        .withEncryptionKeyMetadata(ByteBuffer.wrap("Trino".getBytes(UTF_8)))
+                        .build();
+        table.newAppend().appendFile(dataFile).commit();
+        writeEqualityDeleteToNationTable(table);
+        assertQuery(
+                "SELECT " +
+                        "content, " +
+                        "file_format, " +
+                        "record_count, " +
+                        "CAST(column_sizes AS JSON), " +
+                        "CAST(value_counts AS JSON), " +
+                        "CAST(null_value_counts AS JSON), " +
+                        "CAST(nan_value_counts AS JSON), " +
+                        "CAST(lower_bounds AS JSON), " +
+                        "CAST(upper_bounds AS JSON), " +
+                        "key_metadata, " +
+                        "split_offsets, " +
+                        "equality_ids " +
+                        "FROM \"" + tableName + "$files\"",
+                "VALUES " +
+                        "(0, 'ORC', 25L, null, JSON '{\"1\":25,\"2\":25,\"3\":25,\"4\":25}', JSON '{\"1\":0,\"2\":0,\"3\":0,\"4\":0}', null, " +
+                        "JSON '{\"1\":\"0\",\"2\":\"ALGERIA\",\"3\":\"0\",\"4\":\" haggle. careful\"}', " +
+                        "JSON '{\"1\":\"24\",\"2\":\"VIETNAM\",\"3\":\"4\",\"4\":\"y final packaget\"}', null, null, null), " +
+                        "(0, 'ORC', 10L, JSON '{\"1\":2,\"2\":3}', JSON '{\"1\":5,\"2\":3,\"3\":2}', JSON '{\"1\":0,\"2\":2}', JSON '{\"4\":1}', JSON '{\"1\":\"0\"}', " +
+                        "JSON '{\"1\":\"4\"}', X'54 72 69 6e 6f', ARRAY[4L], null)");
```

Comment:
Can we wrap lines per column? I can't understand the mapping between column and value quickly.